### DISCUSSION TITLE
contribute adding config for Elegoo Neptune 4MAX

### DISCUSSION THREAD

**Discussion by eafq78** [OrcaSlicer/OrcaSlicer#2455] [Q&A]
IDK how to send this to the PR says I need to be a collaborator to create one PR 
This are tested in my printer:
[Elegoo Neptune 4 MAX 0.4 Nozzle.json](https://github.com/SoftFever/OrcaSlicer/files/13032046/Elegoo.Neptune.4.MAX.0.4.Nozzle.json)
[Elegoo Neptune 4 MAX 0.6 nozzle.json](https://github.com/SoftFever/OrcaSlicer/files/13032047/Elegoo.Neptune.4.MAX.0.6.nozzle.json)

**Comment by oconnellrv** [comment]
Cant help on your question but where did you find a .6mm nozzle for the max?

**Reply by eafq78** [reply]
I drill the original one, and is working perfectly
